app/models/site_upload.rb

```diff
@@ -19,7 +19,15 @@
 class SiteUpload < ApplicationRecord
   include Attachmentable
 
+  FAVICON_SIZES = [16, 32, 48].freeze
+  APPLE_ICON_SIZES   = [57, 60, 72, 76, 114, 120, 144, 152, 167, 180, 1024].freeze
+  ANDROID_ICON_SIZES = [36, 48, 72, 96, 144, 192, 256, 384, 512].freeze
```

Comment:
These sizes were repeated in multiple places and in case someone wanted to update this list, they'd have to do it in multiple places.